processor/ratelimitprocessor/gubernator.go

```diff
@@ -115,7 +115,7 @@ func (r *gubernatorRateLimiter) RateLimit(ctx context.Context, hits int) error {
 	}
 
 	if isUnderLimit := resp.GetStatus() == gubernator.Status_UNDER_LIMIT; !isUnderLimit {
-		// TODO add configurable behaviour for returning an error vs. delaying processing.
+		// TODO support `throttle_behavior` config for returning an error vs. delaying processing.
```

Comment:
[to reviewer] I haven't added support for Gubernator in this PR as I'm not sure if we can do it properly without polling. Lmk if this is a blocker, or whether this PR can be merged as-is.